spec/Checker/InvoiceCustomerRelationCheckerSpec.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Sylius\InvoicingPlugin\Checker;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Customer\Context\CustomerContextInterface;
+use Sylius\InvoicingPlugin\Checker\InvoiceCustomerRelationCheckerInterface;
+use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
+use Sylius\InvoicingPlugin\Exception\InvoiceNotAccessible;
+use Sylius\InvoicingPlugin\Repository\InvoiceRepository;
+
+final class InvoiceCustomerRelationCheckerSpec extends ObjectBehavior
+{
+    function let(
+        InvoiceRepository $invoiceRepository,
+        OrderRepositoryInterface $orderRepository
+    ): void {
+        $this->beConstructedWith($invoiceRepository, $orderRepository);
+    }
+
+    function it_implements_invoice_customer_relation_checker_interface(): void
+    {
+        $this->shouldImplement(InvoiceCustomerRelationCheckerInterface::class);
+    }
+
+    function it_checks_if_customer_id_from_order_is_equal_to_customer_id_from_customer_context(
```

Comment:
`it_checks_if_customer_id_from_order_is_equal_to_passed_customer_id`? We're not using context here :)